dendrite/pipeline.yml

```diff
@@ -35,6 +35,24 @@ steps:
       - docker#v3.0.1:
           image: "golang:1.13"
           mount-buildkite-agent: false
+
+  - command:
+      - "curl -fsSL https://get.docker.com -o get-docker.sh"
+      - "sh get-docker.sh"
+      - "docker build -t complement-dendrite -f build/scripts/Complement.Dockerfile ."
+      - "wget https://github.com/matrix-org/complement/archive/master.tar.gz && tar -xzf master.tar.gz"
```

Comment:
would it be possible to build a complement docker image, which includes both the docker commandline and a built (or at least ready-to-build) complement, so we don't need to do this messing about?

Reply:
Yes that's possible.

Reply:
This is actually a bit of a faff to do. I want Complement to be built from `master` every time, and I want that build to include the Docker CLI so I can issue `docker build` on Dendrite.

We're debating where to put docker images full stop so for now I think I'll leave this and we can optimise it later.

Reply:
docker images are still going on dockerhub for now.

Can you at least give us an image so that we don't need to do the awful get-docker thing?

Reply:
Sure, I'll upload one with Go/Docker CLI installed already.

Reply:
Done, which shaves 1min off the runtime.